visual-tests/tests/js-only/text-truncate/text-truncate-column.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { test } from '../../../src/test-runner';
```

Comment:
We can save screenshots by doing only one test, for example, `text-truncate-all.spec.ts`. Other tests are related to the settings-layers mechanism (cell, columns, table), which are so well tested that there is no need to make any more - you are sure that it will work for `cell` and `columns`.